src/topp/QualityControl.cpp

```diff
@@ -74,6 +76,7 @@ class TOPPQualityControl : public TOPPBase
     setValidFormats_("in_con", {"fasta"});
     //possible additions:
     //"mzData,mzXML,dta,dta2d,mgf,featureXML,consensusXML,idXML,pepXML,fid,mzid,trafoXML,fasta"
+    registerFlag_("MS2_id_rate:force_no_fdr", "forces the metric to run if fdr was not made");
```

Comment:
add `(accept all pep-ids as target hits)`??
The default behaviour should also be added to the documentation of compute():
`Only pep-ids with FDR metavalue annotation as 'target' are counted, unless force_fdr flag is set (counts all pep-ids as target)`